packages/mobile/e2e/scripts/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { E2E_TEST_WALLET } from './consts'
+export const Web3 = require('web3')
+export const ContractKit = require('@celo/contractkit')
+export const dotenv = require('dotenv')
```

Comment:
Any reason not to let other modules import these directly?

Also with a `require` we won't get the types.

Reply:
Fixed in [c3d9135](https://github.com/valora-inc/wallet/pull/2325/commits/c3d9135f93a845d1065db1d174e274cee86d96a5)!